//__________________________________________________________________
//Java classes/objetos

import java.util.Scanner;

//01 - Create a class called MyClass.
public class MyClass {

//    02 - Create an object of MyClass called myObj.
//        MyClass myObj = new MyClass();

//    03 - Use myObj to access and print the value of the x attribute of MyClass.
//        int x = 5;
//    public static void main(String[] args) {
//        MyClass myObj = new MyClass();
//        System.out.println(myObj.x);
//    }

//    04 - Call myMethod on the object.
//    public void myMethod() {
//        System.out.println("Hello World");
//    }
//    public static void main(String[] args) {
//        MyClass myObj = new MyClass();
//        myObj.myMethod();
//    }

//    05 - Create and call a class constructor of MyClass
//         Follow the comments to insert the missing parts of the code below:
//    int x;
//    public MyClass() {
//        x = 5;
//    }

//    public static void main(String[] args) {
//        MyClass myObj = new MyClass();
//        System.out.println(myObj.x);
//    }

//    06 - The class below should not be inherited by other classes. Add the correct modifier:
//    final class MyClass(){}

//    07 - Fill in the missing parts to import the java.util.Scanner class from the Java API:
//    import java.util.Scanner;

//    08 - The Car class should inherit the attributes and methods from the Vehicle class. Add the correct keyword
//    to make this possible.
//    class Car extends Vehicle;
//__________________________________________________________________
    //Java exceptions
//    01 - Insert the missing parts to handle the error in the code below.
//    try {
//        int[] myNumbers = {1, 2, 3};
//        System.out.println(myNumbers[10]);
//    } catch (Exception e) {
//        System.out.println("Something went wrong.");
//    }

//    02 - Insert the missing keyword to execute code, after try..catch, regardless of the result.
//    try {
//        int[] myNumbers = {1, 2, 3};
//        System.out.println(myNumbers[10]);
//    } catch (Exception e) {
//        System.out.println("Something went wrong.");
//    } finally {
//        System.out.println("The 'try catch' is finished.");
//    }
}
